libs/js-sdk/src/client.ts

```diff
@@ -65,9 +63,10 @@ export function createBrowserClient({
 
     // Ensure that the init promise doesn't cause an unhandled promise rejection
     initialisedState.initialisedPromise.promise.catch(() => {})
+
     const state = new EffectiveFeaturesState(
         audiences,
-        store || new MemoryEffectiveFeatureStore(initialValues),
+        new MemoryEffectiveFeatureStore(externalStateStore),
```

Comment:
Rather than passing the external state store into the FeatureStore, the SDK could subscribe to changes to the `state` which then writes to the external state store?

This is just an idea for us to discuss, not to necessarily change

Reply:
After discussion we decided to not do this.